packages/omezarr/src/zarr/loading.ts

```diff
@@ -179,10 +179,27 @@ export function pickBestScale(
     }, datasets[0]);
     return choice ?? datasets[datasets.length - 1];
 }
-
+// TODO this is a duplicate of indexOfDimension... delete one of them!
```

Comment:
Let's take care of this unification now. `indexOfDimension` isn't exported in the `index` file (and we don't use it outside the package anyway) so pick which name you like best and we'll go with it!

Reply:
done